internal/component/otelcol/receiver/influxdb/influxdb.go

```diff
@@ -2,39 +2,39 @@
 package influxdb
 
 import (
-    "fmt"
-
-    "github.com/grafana/alloy/internal/component"
-    "github.com/grafana/alloy/internal/component/otelcol"
-    otelcolCfg "github.com/grafana/alloy/internal/component/otelcol/config"
-    "github.com/grafana/alloy/internal/component/otelcol/receiver"
-    "github.com/grafana/alloy/internal/featuregate"
```

Comment:
It looks like the previous committer used spaces instead of tabs and my code formatted did this. Let me know if you want me to revert this change.

Reply:
Its fine.